core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1835,7 +1835,7 @@ object SnappySession extends Logging {
   private[this] val ID = new AtomicInteger(0)
   private[sql] val ExecutionKey = "EXECUTION"
 
-  def tokenizationDisabled: Boolean = java.lang.Boolean.getBoolean("DISABLE_TOKENIZATION")
+  var tokenize: Boolean = true
```

Comment:
should be initialized like for planCaching: Property.Tokenize.get(sessionState.conf)

Reply:
cal also be made private[sql] like for planCaching

Reply:
>> should be initialized like for planCaching: Property.Tokenize.get(sessionState.conf)

This is more like a global variable. No session is attached to it at the time of initialization. So kept it as "true". May be this property should be in global sqlconf.
Is there one like that?

Reply:
Oh, ok. Then Property.Tokenize is better be a SparkConf Val and not SQLVal. Then in SnappyContext.initGlobalSparkContext you could set it: SnappySession.tokenize = Property.Tokenize.get(sc.conf)